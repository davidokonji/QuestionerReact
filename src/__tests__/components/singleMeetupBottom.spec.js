import React from 'react';
import { shallow } from 'enzyme';
import SingleMeetupBottom from '../../components/singleMeetupBottom';

describe('Single Meetup Top Component', () => {
  it('should render component correctly', () => {
    const wrapper = shallow(<SingleMeetupBottom />);
    expect(wrapper).toMatchSnapshot();
  });
});
