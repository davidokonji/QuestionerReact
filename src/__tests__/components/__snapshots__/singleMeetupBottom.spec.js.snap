// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Single Meetup Top Component should render component correctly 1`] = `
<div
  className="py-5"
>
  <div>
    <h2>
      Questions
    </h2>
  </div>
  <div
    className="py-5"
  >
    <p>
       You have no Question 
    </p>
  </div>
</div>
`;
